tests/drivers/flash_api/src/main.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2021-2024 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <zephyr/drivers/flash.h>
+#include <zephyr/device.h>
+
+/** Test instrumentation **/
+/*
+ * Below structure gathers variables that may be used by simulate API calls to mock behaviour
+ * of a flash device; the variables may be used however it is desired, because the it is
+ * up to mocked function and check afterwards to relate a variable to result
+ */
+static struct {
+	/* Set to value returned from any API call */
+	int ret;
+	/* Some size */
+	uint64_t size;
+	/* Allowed offset alignment */
```

Comment:
is this line above a left over?

Reply:
Yes, because when this API test has been created in 2021 it has far more things tested, but I have reduced the scope to be able to make the thing reviewable.

Reply:
Fixed